p7zip/README.md

```diff
@@ -0,0 +1,49 @@
+# p7zip
+
+This is the p7zip compression utility that can be used with Unikraft bincompat mode.
+
+## Content
+
+- `lib/` & `lib64/`: required libs for p7zip
+- `/7z`: the 7z main binary
+- `7z.so`, `7za`, `7zCon.sfx`, `7zr`: different code modules for p7zip that will be used by 7z binary to perform different operations.
+
+## Extracting Dynamic Libraries
+
+```console
+../extract.sh ./7z
+```
+
+## Running on Linux
+
+This will create a new archive called `archive.zip` and add the file `README.md` to it.
+
+```console
+7z a archive.zip README.md
+```
+
+This will extract the archive:
+
+```console
+7z x archive.zip
+```
+
+## Running on Unikraft
+
```

Comment:
Add a line on using the `run.sh` script from the `run-app-elfloader` repository (see an example [here](https://github.com/unikraft/dynamic-apps/tree/master/lang/c/http_server#run-with-unikraft)).